tests/spec/core/issues-notes-spec.js

```diff
@@ -126,6 +126,38 @@ describe("Core — Issues and Notes", function () {
     );
   });
 
+  it("shows labels for github issues", async () => {
+    const githubConfig = {
+      github: "https://github.com/mock-company/mock-repository",
+      githubAPI: `${window.location.origin}/tests/data`,
+    };
+    const ops = {
+      config: githubConfig,
+      body:
+        makeDefaultBody() +
+        `
+        <div class='issue' data-number='1540'>issue is open on github</div>
+        <div class='issue' id='this-is-404' data-number='404'>this is 404</div>
+        <section id='issue-summary'></section>
+        `,
+    };
+    const doc = await makeRSDoc(ops);
+    const issueDiv = doc.getElementById("issue-1540");
+    expect(issueDiv).toBeTruthy();
+    const labels = doc.getElementsByClassName("respec-gh-label");
+    expect(labels[0].innerText).toEqual("refactor");
```

Comment:
nit: it's nice to name each label, so we can then more easily know what which is which. So:

```JS
const [refactorLabel, bugLabel, blankLabel] = doc.querySelectorAll(
  ".respec-gh-label"
);
expect(bugLabel.innerText).toEqual("bug");

```